interfaces/services/block_storage.proto

```diff
@@ -81,6 +81,7 @@ message GetLastCommittedBlockHeightOutput {
 
 message ValidateBlockForCommitInput {
     protocol.BlockPairContainer block_pair = 1;
+    protocol.BlockPairContainer prev_block_pair = 2;
```

Comment:
Which fields are requried? only referenceTime?

Reply:
This one is consistent with existing flow: ValidateBlockForCommit =>  HandleBlockConsensus and propagates all the way to LeanHelix repo code and requires a thorough change...